_modules/dir.test.js

```diff
@@ -0,0 +1,79 @@
+/**************************************************************************************************
+ * Imports
+**************************************************************************************************/
+
+const dir   = require("./dir");
+const faker = require("faker");
+const fs    = require("fs");
+const shell = require("shelljs");
+
+describe("dir", () => {
+    let mockDir;
+    beforeEach(() => {
+        mockDir = faker.random.word();
+    });
+
+    /**************************************************************************************************
+     * deleteIfExists
```

Comment:
Recommend use of our #region / #endregion based comments

Reply:
Can do! I know when this was originally being built out/ported over, we were still kind of on the fence about which style to use as a team. I thinking about raising an issue to flesh out more of a style guide so there's less uncertainty.